plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -34,361 +34,353 @@
 #include <cstdlib>
 
 // reflect chainbase::environment for --print-build-info option
-FC_REFLECT_ENUM( chainbase::environment::os_t,
-                 (OS_LINUX)(OS_MACOS)(OS_WINDOWS)(OS_OTHER) )
-FC_REFLECT_ENUM( chainbase::environment::arch_t,
-                 (ARCH_X86_64)(ARCH_ARM)(ARCH_RISCV)(ARCH_OTHER) )
-FC_REFLECT(chainbase::environment, (debug)(os)(arch)(boost_version)(compiler) )
+FC_REFLECT_ENUM(chainbase::environment::os_t,
```

Comment:
It appears this PR nearly entirely reformats this file which is unacceptable.  If there were relevant changes to this file please make them as tersely as possible otherwise ensure the PR does not mutate it.

Reply:
Of course, I always try to not include any formatting change, I assume it's the `ouuppss` note :) I'll revert all this for sure.